### ISSUE TITLE
Latest is broken on erica push to cloudant

### ISSUE THREAD

**Issue by lancecarlson** [benoitc/erica#102]
On erica version 0.5.0 I get an error:

```
$ erica push https://xxx:yyy@xxx.cloudant.com/db
==> madmin-products (push)
ERROR: push failed while processing /home/lance/Projects/madmin-products: {'EXIT',{{badmatch,{error,forbidden}},
         [{erica_util,db_from_string,2,
                      [{file,"src/erica_util.erl"},{line,74}]},
          {erica_push,push2,3,[{file,"src/erica_push.erl"},{line,51}]},
          {erica_push,push1,3,[{file,"src/erica_push.erl"},{line,44}]},
          {erica_core,run_modules,4,[{file,"src/erica_core.erl"},{line,98}]},
          {erica_core,execute,4,[{file,"src/erica_core.erl"},{line,68}]},
          {erica,main,1,[{file,"src/erica.erl"},{line,12}]},
          {escript,run,2,[{file,"escript.erl"},{line,747}]},
          {escript,start,1,[{file,"escript.erl"},{line,277}]}]}}
```

On my other machine with erica version 0.4.0 it pushes without any issues with the same credentials and database. Any ideas?

**Comment by DamonOehlman** [comment]
I just hit this too, I'll try reverting to `0.4.1` and see if everything works as per @lancecarlson's experience

**Comment by benoitc** [comment]
fixed in 0d439f98bde9ea9c6e7656067700ff45ac26935c
